ai_genomics/getters/openalex.py

```diff
@@ -136,3 +137,15 @@ def work_abstracts(discipline: str, years: List) -> Dict:
             for year in years
         ],
     )
+
+
+def get_openalex_ai_genomics_works() -> pd.DataFrame:
+    """Returns dataframe of AI in genomics OpenAlex works"""
+    try:
+        return pd.read_csv(
+            f"{PROJECT_DIR}/outputs/ai_genomics_provisional_dataset.csv", index_col=0
+        ).drop(columns=["index"])
```

Comment:
If `index` will always be dropped then best to incorporate this into the pipeline that produces the data too.